scripts/ufl-convert

```diff
@@ -38,30 +39,36 @@ def runcmd(cmd):
         print(output)
         sys.exit(-1)
 
+
 def write_file(filename, text):
     "Write text to a file and close it."
     with io.open(filename, "w", encoding="utf-8") as f:
-        f.write(text)
+        f.write(text.decode('utf-8'))  # Python 2
```

Comment:
This feels a bit back and forth. You decode a byte stream to a string according to UTF8, and write that string into a text file encoded in UTF8. Why not just open the file in binary mode and write `text` as it is?